### DISCUSSION TITLE
About Arduino IDE 2.0 beta 4

### DISCUSSION THREAD

**Discussion by rickygai** [earlephilhower/arduino-pico#87] [Ideas]
hi, any plan to support Arduino IDE 2.0 beta 4 as the link below seems not showing anything on the board listing of Arduino IDE 2.0 beta 4:

https://github.com/earlephilhower/arduino-pico/releases/download/global/package_rp2040_index.json

Please advise.

**Comment by blurfl** [comment]
@bodmer posted two nice examples in the [Multicore](https://github.com/earlephilhower/arduino-pico/discussions/83) discussion topic of this discussion.

**Reply by rickygai** [reply]
thanks, I just noticed it, forgotten to change the title, apologizing.

**Reply by earlephilhower** [reply]
Release 1.0.3 installs on the Arduino 2.0 beta.

**Reply by rickygai** [reply]
ok, now it is listed there and installed, Super Cool!

![image](https://user-images.githubusercontent.com/15626855/114207635-2a100c80-998f-11eb-99ea-d5779fdefd69.png)

Thank you :o)

**Reply by earlephilhower** [reply]
Please verify or re-enter the URL in the File->Preferences->Board Manager.   Might need to restart the IDE.

I just entered the URL on a completely new computer after just d/ling and installing the beta and it appeared tout-de-suite...

![image](https://user-images.githubusercontent.com/11875/114207447-3c4b6580-9911-11eb-98c8-9eafb8bc5459.png)

![image](https://user-images.githubusercontent.com/11875/114207545-55541680-9911-11eb-9c39-78e3de6f9aaa.png)

**Reply by earlephilhower** [reply]
Ah, cross-posted.  No problem, glad to see its working!
